pkgs/clightning-plugins/get-sha256.sh

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p git
+set -euo pipefail
+
+archive_hash () {
+    repo=$1
+    rev=$2
+    nix-prefetch-url --unpack "https://github.com/${repo}/archive/${rev}.tar.gz" 2> /dev/null | tail -n 1
+}
+
+TMPDIR="$(mktemp -d -p /tmp)"
+trap "rm -rf $TMPDIR" EXIT
+
+cd "$TMPDIR"
+echo "Fetching latest DavHau/mach-nix release"
```

Comment:
Why are we still calculating the sha256 for `mach-nix`?

Reply:
My bad!  This is an oversight.